waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -76,11 +76,11 @@ template initializedGuard(g: OnchainGroupManager): untyped =
 
 
 proc setMetadata*(g: OnchainGroupManager): RlnRelayResult[void] =
-  if g.latestProcessedBlock.isNone():
+  if g.latestProcessedBlock == 0:
```

Comment:
mmm do we still need to error here? guess 0 could be an ok value? eg at startup?

Reply:
there shouldn't be a reason to save the metadata if the latest block number is 0, hence there were 2 options here, either a noop or an error, went for the latter